### DISCUSSION TITLE
Convert CityEnviron to Linux?

### DISCUSSION THREAD

**Discussion by Steinmar12** [microsoft/AirSim#3842] [Support Q&A]
Hello,

i'm new to UE4 and Airsim, so sorry for the noob question.
I would like to use your CityEnviron environment but from what I understand it's made to be use on Windows, is there a way to convert or adapt it to Linux?

**Comment by zimmy87** [comment]
Not currently, the CityEnviron environment currently has dependencies that prevent us from building a Linux release with it.
